core/launcher/src/manager/app_launcher.rs

```diff
@@ -608,9 +608,13 @@ impl AppLauncher {
             }
         }
 
-        let modified_url = Self::get_modified_url(&manifest, &sessionid);
-        debug!("modified url with sessionid : {:?}", modified_url);
-        Ok(modified_url)
+        if !sessionid.is_empty() {
```

Comment:
needs merging with latest code.

Reply:
Yes, the branch was behind the main. Now Merged it.